src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,26 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private static final UserService AUTHENTICATION_SERVICE = new UserService();
```

Comment:
```suggestion
    private final UserService userService = new UserService();
```
it's not a constant, and shouldn't be one - we just inject this dependency